### ISSUE TITLE
Crash because of sentinels.json:15:25: missing mandatory member "hit_size_relative"

### ISSUE THREAD

**Issue by NetSysFire** [CleverRaven/Cataclysm-DDA#53494]
### Describe the bug

See title.

### Steps to reproduce

1. Build latest master on linux (or use the snapshot which was just built).
2. Launch it.
3. Crash

```
MESSAGE: Json error: usr/share/cataclysm-dda/core/sentinels.json:15:25: missing mandatory member "hit_size_relative"

    "limb_type": "other",
    "hit_size": 0,
    "hit_difficulty": 0,
                        ^
    "side": "both",
    "base_hp": 60,
    "drench_capacity": 1,

STACK TRACE:

    cataclysm-tiles(debug_write_backtrace(std::ostream&)+0x46) [0x561720d16e00]
    cataclysm-tiles(+0x9eed72) [0x561720cf4d72]
    cataclysm-tiles(+0x9eef60) [0x561720cf4f60]
    /usr/lib/libstdc++.so.6(+0xa5c8a) [0x7fc1b0b0dc8a]
    /usr/lib/libstdc++.so.6(+0xa5cf7) [0x7fc1b0b0dcf7]
    /usr/lib/libstdc++.so.6(+0xa5f8e) [0x7fc1b0b0df8e]
    cataclysm-tiles(DynamicDataLoader::load_data_from_path(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, loading_ui&)+0x306) [0x561720f30014]
    cataclysm-tiles(game::load_core_data(loading_ui&)+0x67) [0x561720e3df6b]
    cataclysm-tiles(main_menu::init_strings()+0x708) [0x561721134bae]
    cataclysm-tiles(main_menu::opening_screen()+0x9d) [0x561721139b53]
    cataclysm-tiles(main+0x173e) [0x56172090065c]
    /usr/lib/libc.so.6(__libc_start_main+0xd5) [0x7fc1b0764b25]
    cataclysm-tiles(_start+0x2e) [0x561720a2a62e]

    Attempting to repeat stack trace using debug symbols…
    debug_write_backtrace(std::ostream&)
    ??:?
    init_crash_handlers()
    ??:?
    init_crash_handlers()
    ??:?
    ??
    ??:0
    ??
    ??:0
    ??
    ??:0
    DynamicDataLoader::load_data_from_path(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, loading_ui&)
    ??:?
    game::load_core_data(loading_ui&)
    ??:?
    main_menu::init_strings()
    ??:?
    main_menu::opening_screen()
    ??:?
    main
    ??:?
    __libc_start_main
    ??:?
    _start
    ??:?
```

### Expected behavior

No crash.

### Versions and configuration

- very latest experimental as of 4ac5f7158aefb817e286c7dac95d68e252889e8a

**Comment by NetSysFire** [comment]
It works again if I re-add `"hit_size_relative": [ 0, 0, 0 ],` to the file, which was removed in dd5ae68efb6a62ac3edf144226b9e08ddd1c98f3.

Ping @Venera3?

**Comment by NetSysFire** [comment]
It appears even if that fixed the initial issue, there are still more which prevent you from progressing further than the title screen:

```
 DEBUG    : Error: Json error: usr/share/cataclysm-dda/json/body_parts.json:20:25: missing mandatory member "hit_size_relative"

    "is_vital": true,
    "hit_size": 36,
    "hit_difficulty": 1,
                        ^
    "side": "both",
    "legacy_id": "TORSO",
    "stylish_bonus": 6,


 FUNCTION : bool main_menu::load_character_tab(bool)
 FILE     : src/main_menu.cpp
 LINE     : 1069
 VERSION  : 0.F
```

**Comment by Venera3** [comment]
That's...interesting. `hit_size_relative` doesn't exist as a bodypart field anymore, and I've been using a build with that change without incident yesterday.

**Comment by Venera3** [comment]
I can't reproduce on a fresh master or on ` cdda-windows-tiles-x64-msvc-2021-12-16-1453 `.

**Comment by NetSysFire** [comment]
I actually inspected the raw binary file and found `hit_size_relative` as a string in that release for the linux release and in the binary that I built. It is not present anymore in the newest release but I can't run that because of `error while loading shared libraries: libncursesw.so.5: cannot open shared object file: No such file or directory` since I have ncurses 6.3.

If the binary, which I build from a fresh master without any artifacts from builds before, still contains that string I officially believe in ghosts.
